### ISSUE TITLE
Amphetamine/dextroamphetamine dependence

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6537]
**Preferred term label:** Amphetamine/dextroamphetamine dependence

**Synonyms**
Amphetamine/dextroamphetamine  addiction 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Stimulants dependence (to be created)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
